mailet/standard/src/test/java/org/apache/james/transport/matchers/HostIsLocalTest.java

```diff
@@ -46,7 +46,7 @@
 
     @Before
     public void setUp() throws Exception {
-        mockedMail = new FakeMail();
+        mockedMail = FakeMail.builder().build();
```

Comment:
Tried to make FakeMail full support builder (which means no setting outside of builder) but a lot of side effect